ClientLib/src/main/java/com/o3dr/services/android/lib/drone/mission/MissionItemType.java

```diff
@@ -113,6 +114,18 @@ public MissionItem getNewItem() {
         }
     },
 
+    DO_LAND_START("Do Land start") {
```

Comment:
@daimonkor for the front end clients running on top of 3DR Services. They need a way to distinguish mission items per vehicle type. For example, it doesn't make sense to show the user a `DO_LAND_START` mission item in the editor tools in Tower if he's building a mission for a copter.

And in the future, there's likely to be more mission items specific to a vehicle type. I'd consider it a bad implementation if we have no way to distinguish them besides throwing them at the autopilot and seeing what sticks.